tweetapi/views.py

```diff
@@ -29,11 +27,11 @@ class TweetView(generics.ListCreateAPIView):
     serializer_class = TweetSerializer
 
 class TweetViewDetails(generics.RetrieveDestroyAPIView):
```

Comment:
another user can delete your tweet

Reply:
no sir you alone can delete your tweet

Reply:
I don't see a permission

Reply:
the permission you just added will not let another user see a tweet

Reply:
that was an oversight i will take it out of the list and create view....